packages/x-date-pickers/src/AdapterLuxon/AdapterLuxon.test.tsx

```diff
@@ -63,7 +68,7 @@ describe('<AdapterLuxon />', () => {
       ) => {
         const date = adapter.date('2020-02-01T23:44:00.000Z')!;
 
-        expect(adapter.format(date, format)).to.equal(expectedWithEn);
+        expect(cleanText(adapter.format(date, format))).to.equal(expectedWithEn);
```

Comment:
Yes, 
I can't find the source again, but the reason of my strange space is that the latest version of Chrome has a small bug that add this space in the `Intl` API instead of a regular space.